components/examples/window/src/main.ts

```diff
@@ -0,0 +1,178 @@
+import van from "vanjs-core"
+import { FloatingWindow, Tabs } from "vanjs-ui"
+
+export type CSSPropertyBag = Record<string, string | number>
+
+const toStyleStr = (style: CSSPropertyBag) =>
+  Object.entries(style).map(([k, v]) => `${k}: ${v};`).join("")
+
+const { button, div, h1, p, a, b, code, pre, span, style } = van.tags
+const example1 = () => {
+  const closed = van.state(false)
+  const x = van.state(100)
+  const y = van.state(100)
+  const width = van.state(300)
+  const height = van.state(200)
+
+  van.add(document.body, FloatingWindow({
+    title: "Example Window 1",
+    closed,
+    x,
+    y,
+    width,
+    height
+  },
+    div({ style: "padding: 1rem;" },
+      p("Hello, World!"),
+      div({ style: "display: flex; justify-content: center;" },
+        button({ onclick: () => closed.val = true }, "Close Window"),
+      ),
+    )
+  ))
+}
+
+const example2 = () => {
+  const closed = van.state(false)
+  const x = van.state(150)
+  const y = van.state(150)
+  const width = van.state(300)
+  const height = van.state(200)
+
+  van.add(document.body, FloatingWindow({
+    title: "Example Window 2",
+    closed,
+    x,
+    y,
+    width,
+    height,
+    closeCross: true,
+    headerStyleOverrides: {
+      "background-color": "lightblue",
+    }
+  },
+    div({ style: "display: flex; justify-content: center;" },
+      p("This is another floating window!"),
+    )
+  ))
+}
+
+const example3 = () => {
+  const closed = van.state(false)
+  const x = van.state(200)
+  const y = van.state(200)
+  const width = van.state(500)
+  const height = van.state(300)
+
+  van.add(document.body, FloatingWindow({
```

Comment:
This example is great! However, there is a minor issue: The height of the cross button is not the same as the title bar of the Tabs. As a result, the cross button appears a little bit below the center: 
![image](https://github.com/vanjs-org/van/assets/132854319/36f6ecac-f07b-4ece-a65e-010109655b9d)

I will appreciate if you can get it fixed.

Reply:
done in 1d11021